repo/blob/s3/s3_options.go

```diff
@@ -23,6 +23,9 @@ type Options struct {
 	SecretAccessKey string `json:"secretAccessKey" kopia:"sensitive"`
 	SessionToken    string `json:"sessionToken"    kopia:"sensitive"`
 
+	// Timeout is used as a deadline context in Minio API calls.
+	RequestTimeout int64 `json:"requestTimeout,omitempty"`
```

Comment:
please rename the field and JSON field as RequestTimeoutSeconds

Reply:
Done